src/platform/ASR/PlatformManagerImpl.cpp

```diff
@@ -93,47 +91,86 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
 
 void PlatformManagerImpl::_RunEventLoop()
 {
-    kTaskRunningEventFlag = 1;
-
+    mRunningEventActive = true;
```

Comment:
I'm a little confused about this variable:

1. It should be documented in terms of what it means.
2. Why do we hold `mEventMutex` some times, but not others, when accessing this variable?

Reply:
if don't think about the queue and mutex variables, we can remove mRunningEventActive to reduce complexity, after all, they take memory little...